pkg/render/common/elasticsearch/service.go

```diff
@@ -7,8 +7,8 @@ import (
 )
 
 const (
-	httpsEndpoint     = "https://tigera-secure-es-http.tigera-elasticsearch.svc:9200"
-	httpsFQDNEndpoint = "https://tigera-secure-es-http.tigera-elasticsearch.svc.%s:9200"
+	httpsEndpoint     = "https://tigera-secure-es-gateway-http.tigera-elasticsearch.svc:9200"
```

Comment:
Does this mean that there was some agreement reached that we're going to require customers with managed clusters to upgrade those clusters when they upgrade the management cluster?

Reply:
The customers won't need to upgrade the managed clusters, they will still have the old code running meaning that their components will still be configured to hit the old service. The new certificate passed down to these components will contain `tigera-secure-es-http` in the SANs field in order to support these older clusters/service names. Voltron will be configured to forward requests to the new es-gateway service. I've verified that this works by testing on 3.7 managed cluster connected to a custom management cluster running this operator image.

Reply:
OK that sounds like a good plan.